.devcontainer/Dockerfile.ci

```diff
@@ -2,7 +2,7 @@ FROM adrienaury/go-devcontainer-ci:v0.5
 
 USER root
 
-RUN apk add --update --progress --no-cache make gomplate
+RUN apk add --update --progress --no-cache make gomplate jq

```

Comment:
Try to use different version of devcontainer

https://github.com/adrienaury/go-devcontainer/issues/3#issuecomment-1310164891